src/proj.h

```diff
@@ -341,6 +341,9 @@ void proj_errno_restore (PJ *P, int err);
 
 char *proj_get_definition (PJ *P);
 
+/* deallocate anything obtained from PROJ.4 */
```

Comment:
Down at the library level it is not exactly a getter (that could just be the case of returning P->params) ... "create" is much in accordance with the actual construction work done by pj_get_def: It actually walks the P->param list, element by element expanding the return string.

Now, the user does not need to know this, but for reasons of communication, I believe it is a good thing to have the reminder that "what you get by a constructor, you get rid of by a destructor"